### PR TITLE
[mtdev] Switch to git submodule and update to 1.1.6. JB#56424

### PR DESCRIPTION
**Pull request by pvuorela**
@mlehtima @Tomin1

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[submodule "upstream"]
+	path = upstream
+	url = http://bitmath.org/git/mtdev.git
```
Comment (Tomin1) [inline .gitmodules]: I would ask for a mirror for this.

**Review by Tomin1 — APPROVED** [review]
Looks fine but still would use a mirror here.

File: .gitmodules (hunk 1 above)
Comment (pvuorela) [reply .gitmodules]: Yea, could make sense here. Asked. I'll assume replacing this with one is still approved.

Comment (Tomin1) [reply .gitmodules]: Yes.

**Pull request merged**